addons/web/static/src/core/registry.js

```diff
@@ -11,12 +11,18 @@ export class DuplicatedKeyError extends Error {}
 // Validation
 // -----------------------------------------------------------------------------
 
-const validateSchema = (value, schema) => {
+const validateSchema = (name, key, value, schema) => {
     if (!odoo.debug) {
         return;
     }
-    validate(value, schema);
-}
+    try {
+        validate(value, schema);
+    } catch (error) {
+        throw new Error(
+            `Cannot add key "${key}" in the "${name}" registry: it has a validation error : ${error}`
```

Comment:
weird message in a validateSchema functio: we're not necessarily adding a key in a registry here. As a matter of fact, we can add a schema on an already populated registry
=> suggestion:
`Validation error for key "${key}" in registry "${name}": ${error}`